classes/ActionScheduler_WPCLI_Scheduler_command.php

```diff
@@ -80,11 +84,14 @@ public function run( $args, $assoc_args ) {
 	 *
 	 * @param int $total
 	 */
-	protected function print_total_actions( $total ) {
+	protected function print_total_actions( $total, $print_timestamp = false ) {
+		$timestamp = $print_timestamp ? as_get_datetime_object()->getCliTimestamp() : '';
+
 		WP_CLI::log(
 			sprintf(
 				/* translators: %d refers to how many scheduled taks were found to run */
-				_n( 'Found %d scheduled task', 'Found %d scheduled tasks', $total, 'action-scheduler' ),
+				_n( '%sFound %d scheduled task', '%sFound %d scheduled tasks', $total, 'action-scheduler' ),
```

Comment:
`ActionScheduler_WPCLI_QueueRunner` adds the date/time outside the i18n function. This class is adding them _within_ the i18n functions. I think including them outside is the better approach. But regardless of which approach is taken, it should be consistent.

Reply:
Yep, missed this one; fixed.